.github/workflows/clang-linux-test-x64.yml

```diff
@@ -0,0 +1,125 @@
+name: AscEmu CI (clang)
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+
+  Classic:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup Environment
+      run: |
+        sudo apt-get update && sudo apt-get install -y zlib1g-dev
+        sudo apt-get update && sudo apt-get install -y libssl-dev
+        sudo apt-get update && sudo apt-get install -y libpcre3-dev
+        sudo apt-get update && sudo apt-get install -y libbz2-dev
+        sudo apt-get update && sudo apt-get install -y default-libmysqlclient-dev
+        sudo apt-get update && sudo apt-get install -y libmysql++-dev
+    - name: AscEmu Build Classic
+      run: |
+        gcc --version
```

Comment:
Since we are using Clang, use clang --version

Reply:
up